src/test/java/underdogs/devbie/auth/jwt/JwtTokenProviderTest.java

```diff
@@ -0,0 +1,53 @@
+package underdogs.devbie.auth.jwt;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import underdogs.devbie.auth.dto.UserTokenDto;
+import underdogs.devbie.auth.exception.InvalidAuthenticationException;
+import underdogs.devbie.user.domain.User;
+
+class JwtTokenProviderTest {
+
+    private JwtTokenProvider jwtTokenProvider;
+
+    private final Long userId = 1L;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        jwtTokenProvider = new JwtTokenProvider("secretKey", 360000);
+        user = User.builder().id(userId).oauthId("oauthId").email("email").build();
+    }
+
+    @Test
+    @DisplayName("jwt 토큰 생성")
```

Comment:
아직 테스트코드의 컨변센이 없어서 그렇지만..
테스트코드는 명세서라고 생각되서 의견남겨봅니다.
```suggestion
    @Test
    @DisplayName("jwt 토큰 생성한다.")
```

Reply:
개인적으로 @DisplayName은 서술어는 안쓰는게 더 깔끔하다고 생각하는데, 이 부분도 월요일에 논의해보시죠!